app/services/exports/case_log_export_service.rb

```diff
@@ -6,10 +6,14 @@ def initialize(storage_service, logger = Rails.logger)
     end
 
     def export_case_logs
-      case_logs = retrieve_case_logs
-      export = save_export_run
+      current_time = Time.zone.now
+      case_logs = retrieve_case_logs(current_time)
+      export = save_export_run(current_time)
       write_master_manifest(export)
       write_export_data(case_logs)
+      export.save!
+    rescue StandardError => e
```

Comment:
Wonder whether letting this go straight to Sentry might be more useful than just logging since it's likely to mean a run has totally failed and we'd want that to be noisy?

Reply:
I don't think so

Reply:
You could trigger Sentry manually within the rescue block if you wanted to also log it